src/components/Cards/index.tsx

```diff
@@ -1,15 +1,31 @@
-import { useFetch } from "../../utils/hooks/useFetch";
-import { formatDate, generateTimeText } from "../../utils/hooks/date";
-import thumbnail from "../../assets/thumbnail.svg";
-import styles from "./index.module.css";
+import { useFetch } from '../../hooks/useFetch';
+import { formatDate, generateTimeText } from '../../hooks/date';
+import thumbnail from '../../assets/thumbnail.svg';
+import styles from './index.module.css';
 
-function Cards(props) {
-  const CardData = useFetch(props.url);
+interface Link {
+  id: string;
+  url: string;
+  imageSource?: string;
+  title: string;
+  createdAt: Date;
+  description: string;
+}
```

Comment:
프로젝트 전반에 걸쳐 사용되는 타입들은 묶어서 types 폴더에 넣어주셔도 좋을 거 같아요!
실제 사용하실 때는 import 로 가져와서 바로 쓰심 됩니다!
```suggestion
export interface Link {
  id: string;
  url: string;
  imageSource?: string;
  title: string;
  createdAt: Date;
  description: string;
}
```